docs/usage/configuration.rst

```diff
@@ -0,0 +1,312 @@
+                            "foo.baz"
+                        ],
+                        "warning_below": 10.0,
+                        "critical_below": 5.0
+                    }
+                }
+            }
+
+:code:`reporting_host`
+    Name of the host for which check results are reported, as configured in Nagios/Centreon.
+
+    Default:
+        The current hostname returned by :manpage:`gethostname(2)`.
+
+:code:`resend_interval`
+    .. _global_resend_interval:
+
+    Global default resend interval (see :ref:`Check configuration/resend_interval<resend_interval>`).
+
+    Default
+        :literal:`"3min"`
+
+Check configuration
+^^^^^^^^^^^^^^^^^^^
+
+A single check monitors a set of metrics for abnormal behaviour.
+It continuously consumes new data points for these metrics
+and reports and `overall state`: if values of a single metric exceed their :ref:`allowed range<abnormal-ranges>`
+or there are no new values after :ref:`a certain time<timeout>`
+a state of :literal:`WARNING` or :literal:`CRITICAL` is reported.
+
+:code:`metrics` (list of strings)
+    A list of metrics that should be monitored.
+
+    This list is **mandatory** and required to be *non-empty*.
+
+
+:code:`warning_above`, :code:`warning_below`, :code:`critical_above`, :code:`critical_below` (number)
+    .. _abnormal-ranges:
+
+    Range of value which should trigger a :literal:`WARNING` (resp. :literal:`CRITICAL`) status report to be sent.
+    We call the intervals :math:`[-∞, \mathtt{warning\_below}) \cup (\mathtt{warning\_above}, ∞]` the *warning range*;
+    values within that range trigger a :literal:`WARNING`.
+    The *critical range* is defined similarly.
+
+    The critical range must be contained within the warning range, otherwise the configuration is rejected.
+    Explicitly, we required
+
+    .. math::
+        \mathtt{critical\_below}
+            ≤ \mathtt{warning\_below}
+            < \mathtt{warning\_above}
+            ≤ \mathtt{critical\_above}
+
+    Defaults
+        * :math:`-∞` (:code:`{warning,critical}_below`)
+        * :math:`∞`  (:code:`{warning,critical}_above`)
+
+        You cannot put :math:`±∞` directly into the check configuration.
+        Since they are the default anyway, simply omit the relevant key if necessary.
+
+    Example
+        An example check configuration with all ranges specified:
+
+        .. code-block:: json
+
+            {
+                "checks": {
+                    "foo": {
+                        "metrics": [ "foo.bar", "foo.baz" ],
+                        "critical_below":   5.0,
+                        "warning_below":   10.0,
+                        "warning_above":   95.0,
+                        "critical_above": 100.0
+                    }
+                }
+            }
+
+:code:`timeout` (:ref:`duration<Duration>`)
+    .. _timeout:
+
+    Send check result of severity :literal:`WARNING` if values arrive apart more than the specified period.
```

Comment:
Emphasize that the timeout checks alone don't require processing of the messages and thus, can be enabled for metrics with large messages or high sampling rate.